src/test/resources/projects/dependencyResolutionManagement/projmap/build.gradle

```diff
@@ -69,7 +69,7 @@ jar {
 }
 
 minecraft {
-	accessWidener = file("src/main/resources/modid.accesswidener")
+	accessWidener.set file("src/main/resources/modid.accesswidener")
```

Comment:
Fyi, `=` works with properties in the groovy dsl. Also, shouldn't this be `accessWidenerPath`?